### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by kathytbui**
### New Feature Submissions:
* [x] Have you added an explanation of what your changes do?
* [x] Have you linted your code locally prior to submission?

### Testing:
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed?

**Test coverage** 

Test are all passing - 100%
simplecov - 100% coverage
rubocop - 0 offenses
--------------------------------------------------------------------------------
### Change Log:

- Updated specs to fix merge conflicts
- Removed index spec as users dash should go to a show page
- Fix navbar tests to test for logged in vs logged out user
- Welcome index updated to test for login link
- Updated rails helper user mock
- Updated controllers and views to fix merge issues
- Removed dashboard index view - not used
- Added helper method to application controller for current_user
- Update views to match tests
-

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
```
Comment (dtsehl) [inline app/controllers/application_controller.rb:2]: Good catch on this one!

File: spec/features/welcome/index_spec.rb [hunk 2]
```diff
@@ -16,8 +15,7 @@
       within ('.description') do
         expect(page).to have_content('This app allows you to glitch a photo that is created by an AI based on a word or phrase that you provide.') # Just a placeholder - Need an actual description
       end
-      expect(page).to have_css('img[src="https://www.webdesignbites.com/wp-content/uploads/2019/04/glitch-art.jpg"]') #"img[src=\"#{link}\"]"
-      expect(page).to have_button('Login with Google to Start Glitching')
+      expect(page).to have_link('Login with Google to Start Glitching')
```
Comment (dtsehl) [inline spec/features/welcome/index_spec.rb:18]: Turns out Bootstrap makes this a link and not button, even though it looks like a button.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -82,8 +82,8 @@ def mock_user
        :email => "name@name.com"
       },
       :credentials => {
-       :google_token => "google_token",
-       :google_refresh_token => "google_refresh_token"
+       :token => "google_token",
```
Comment (janegreene) [inline spec/rails_helper.rb:85]: Thanks for the fix!

**Review by sciencefixion — APPROVED** [review]
Wonderful work on all those errors!

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -7,25 +7,42 @@
...
-        <ul class="navbar-page-links">
-          <li class="nav">
-            <a class="nav-link" href="/">Home</a>
-          </li>
-        </ul>
-        <ul class="navbar-login-links">
-          <li class="nav">
-            <a class="nav-link" href="/auth/google_oauth2">Login</a>
-          </li>
+    <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #000000;">
```
Comment (janegreene) [inline app/views/layouts/application.html.erb:20]: this is so fancy!

**Pull request merged**